packages/babel-generator/src/generators/flow.js

```diff
@@ -221,11 +221,7 @@ export function TypeAnnotation(node: Object) {
 }
 
 export function TypeParameter(node: Object) {
-  if (node.variance === "plus") {
-    this.token("+");
-  } else if (node.variance === "minus") {
-    this.token("-");
-  }
```

Comment:
You're right, it is a breaking change. I can roll back that part of the change, so the TypeParameter node is unchanged, if you think that's best.

Flow 0.34, which introduces property variance, will also make the same breaking change to the TypeParameter AST.